net/zerotier/files/etc/config/zerotier

```diff
@@ -18,3 +18,6 @@ config zerotier sample_config
 	# Join a public network called Earth
 	list join '8056c2e21c000001'
 	#list join '<other_network>'
+
+	# Allow zerotier clients access your LAN network
+	option nat 0
```

Comment:
-1. NAT rules should be done with firewall zones

Reply:
@zhengmz thank you for your effort. But I have to agree with @LGA1150

Reply:
It is a bit sad to close this MR, so I am thinking if you can instead put the instructions into the wiki. Or maybe change the MR to use the firewall settings in /etc/config/firewall.